pkg/server/storage/mysql.go

```diff
@@ -55,6 +54,29 @@ func (my *MysqlStorage) QueryHostkeys() (ResultIterator, error) {
 	return &SqlResultIterator{Rows: rows}, nil
 }
 
+func (my *MysqlStorage) RecordGithubMapping(mapping map[string]string) error {
+	for ssoIdentity, githubUser := range mapping {
+		_, err := my.DB.Exec(
+			"INSERT INTO github (ssoIdentity, githubUser) VALUES (?, ?) ON DUPLICATE KEY UPDATE githubUser = ?",
```

Comment:
Hm. Isn't your key the actual `Id` (not `ssoIdentity` and not `githubUser`). I'm not sure if that would work well. I wonder if your PrimaryKey should be just `ssoIdentity` but maybe that's against some best practices.

Reply:
that makes sense to me, don't think thats against any best practices maybe @mbyczkowski would know

Reply:
AFAIK, you want to have a primary key, but it doesn't have to be an auto-incrementing one.